README.md

```diff
@@ -106,18 +106,37 @@ cd /path/to/ros_workspace
 catkin_make
 ```
 
+Sunrise Robot Application upload
+------------
+You need a specific application to run on the robot side.
+0. Make sure a Windows laptop is connected on the X66 Ethernet port and has IP `172.31.1.42` mask `255.255.0.0`
+1. Create a new Sunrise project with Sunrise Workbench and setup an empty RobotApplication template 
+2. Setup the safety configuration in `SafetyConfiguration.sconf` ([example](https://github.com/IFL-CAMP/iiwa_stack/wiki/safetyconf))
+3. Replace the empty template with the [online app](https://github.com/epfl-lasa/iiwa_ros/blob/master/iiwa_driver/java/FRIOverlay.java)
+4. In `StationSetup.cat`, tab `Software`, active the FRI extension, push it to the robot with `Installation` and accept the reboot question
+5. Synchronise your new Sunrise project to the robot with icon `Synchronize project`
+6. On the Smartpad tablet, your app must be listed in [Applications] and you must also see a new [FRI] tab
+
 Basic Usage
 --------------
 
 ### Bringup iiwa_driver
 
 **Control IIWA with FRI**
 
-```sh
-roslaunch iiwa_driver iiwa_bringup.launch
-```
+1. Make sure your Linux/ROS laptop is connected on the KONI Ethernet port and has IP `192.170.10.100` mask `255.255.255.0`.
```

Comment:
I have the following questions regarding the IP config:
1. I am not able to ping the robot with [IPv4](https://github.com/epfl-lasa/iiwa_ros/blob/c75f970cae69eb89a4c1e69f63c83824c549dc3d/iiwa_driver/config/iiwa.yaml#L3) and Robot IP as mentioned here.
2. Do we need to change the IP [here](https://github.com/epfl-lasa/iiwa_ros/blob/c75f970cae69eb89a4c1e69f63c83824c549dc3d/iiwa_driver/java/FRIOverlay.java#L36) to Linux /ROS Master
3. I connected a monitor to KONI and checked the IPv4 configuration of two adopters first one 172.31.1.147 (Here I connect my Sunrise to sync the and load the app, etc) and second IPv4 config 192.168.0.2. In my guess, the second adopter is KONI IPv4 config.
4. Also, I am confused about whether to change the KONI IP in setup.cat. Currently, it is set to 192.168.0.7 

Thank you